mbed_greentea/mbed_greentea_cli.py

```diff
@@ -180,6 +183,45 @@ def filter_names_by_prefix(test_case_name_list, prefix_name):
 
     return filtered_ctest_test_list
 
+def create_testlink_test_list(test_list, available_test_cases, filtered_test_cases, test_spec):
```

Comment:
We can use word test pan in greentea but not TestLink. This feature is not specific to TestLink. We should be able to take test plan from any test management system.